lib/hexpm/repository/package.ex

```diff
@@ -115,6 +115,20 @@ defmodule Hexpm.Repository.Package do
     end
   end
 
+  defp search_param("depends", search, query) do
+    dependencies = from(rq in Requirement,
+      join: p in assoc(rq, :dependency),
+      join: rl in assoc(rq, :release),
+      select: rl.package_id,
+      distinct: rl.package_id,
+      where: p.name == ^search,
+    )
+
+    from(p in query,
+      join: d in subquery(dependencies),
```

Comment:
not sure if it's a good idea to to use `subquery` here but `from(p in query, where: id == subquery(dependencies))` isn't valid 🤔.